modules/gdscript/register_types.cpp

```diff
@@ -42,6 +42,7 @@ void register_gdscript_types() {
 
 	ClassDB::register_class<GDScript>();
 	ClassDB::register_virtual_class<GDFunctionState>();
+	ClassDB::register_virtual_class<GDNativeClass>();
```

Comment:
Not sure if `GDNativeClass` is meant to be exposed to scripts...

Reply:
@karroffel ?

Reply:
That has nothing to do with me... This is the type of "the core types" in GDScript.

```gdscript
var clazz = Timer # This has type GDNativeClass
var object = clazz.new()
```

So it actually has nothing to do with GDNative :P I don't think it needs to be exposed to scripts though since that's all done in the GDScript module, not the scripts.

Reply:
Ah, right... my mistake. Should we rename GDNative to DLScript to avoid this confusion?